### PR TITLE
Restyle about dialog

### PR DESCRIPTION
**Pull request by kostyafarber**
# Description
The purpose of the pull request is to improve upon the style of the help dialog in Jupyter Notebook #6552 

The updated dialog is as below.

Screen Shot 2022-10-18 at 17 19 53

### CODE REVIEW

**Comment by jtpio** [comment]
Thanks @kostyafarber this looks good.

I'm currently investigating the CI failure in https://github.com/jupyter/notebook/pull/6579, will try to get it in before merging this PR.

File: packages/help-extension/src/index.tsx [hunk 1]
```diff
@@ -125,7 +125,9 @@ const plugin: JupyterFrontEndPlugin<void> = {
         );
 
         const notebookURL = 'https://github.com/jupyter/notebook';
-        const linkLabel = trans.__('JUPYTER NOTEBOOK ON GITHUB');
+        const contributorURL = 'https://github.com/jupyter/notebook/pulse';
+        const aboutJupyter = trans.__('JUPYTER NOTEBOOK ON GITHUB');
+        const contList = trans.__('CONTRIBUTOR LIST');
```
Comment (jtpio) [inline packages/help-extension/src/index.tsx:130]: Maybe we can rename the variable for better readability?

```suggestion
        const contributorList = trans.__('CONTRIBUTOR LIST');
```

Comment (kostyafarber) [reply packages/help-extension/src/index.tsx:130]: Sure that sounds good! I think that is more informative.

**Comment by kostyafarber** [comment]
@jtpio also I thought of perhaps making the links default to the orange highlight as in:

image

I think it would look better than the purple as it is now. 

Also making the hover state over the url links the same grey as the dismiss button, as in the current JuypterLite:

image

I couldn't figure out how to make those changes 😞 but they could round out the design and make it look better I think.

Just some thoughts!

File: packages/help-extension/src/index.tsx [hunk 2]
```diff
@@ -134,15 +136,25 @@ const plugin: JupyterFrontEndPlugin<void> = {
...
-              {linkLabel}
+              {aboutJupyter}
+            </a>
+            <a
+              href={contributorURL}
+              target="_blank"
+              rel="noopener noreferrer"
+              className="jp-Button-flat jp-AboutNotebook-about-externalLinks"
+            >
+              {contList}
```
[the remark this answers is not in the document]
Comment (kostyafarber) [reply packages/help-extension/src/index.tsx]: Thanks silly mistake by me

**Comment by jtpio** [comment]
> I couldn't figure out how to make those changes disappointed but they could round out the design and make it look better I think.

It's probably in one of the CSS classes, we could investigate in a follow-up PR how to improve it yes.

**Comment by jtpio** [comment]
Looks like the prettier failure is relevant (would need a lint pass): https://github.com/jupyter/notebook/actions/runs/3289225812/jobs/5420503955

**Review by jtpio — APPROVED** [review]
Thanks!

**Comment by jtpio** [comment]
@kostyafarber if you feel like iterating on this don't hesitate to open a new PR, thanks!

**Comment by kostyafarber** [comment]
@jtpio Hey sorry to be that guy but would you be able to add the `hacktoberfest-accepted` label on this PR? Would be much appreciated 😄

**Comment by jtpio** [comment]
Sure thing, done!

**Pull request merged**